src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,55 @@
 import React from 'react';
+import propTypes from 'prop-types';
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ Movie }) => (
+  <div className="card">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          // eslint-disable-next-line max-len
+          src={Movie.imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8">{Movie.title}</p>
```

Comment:
```suggestion
          <p className="title is-8">
            {Movie.title}
          </p>
```
когда есть атрибуты, лучше перенести контент на следующую строку